src/radical/pilot/agent/resource_manager/base.py

```diff
@@ -61,6 +61,7 @@ class RMInfo(ru.TypedDict):
             'mem_per_node'     : int,           # memory per node (MB)
 
             'details'          : {None: None},  # dict of launch method info
+            'sys_arch'         : {str: None},   # system architecture flags
```

Comment:
let's use `details` for this?
```python
'details': {'exact': bool(self._rcfg.get('system_architecture', {}).get('exclusive'))}
```

all other information that we have inside `system_architecture` is used in other `RMInfo` parameters, thus I would go for to extract it and not to preserve the original state, especially `details` is already used for additional data for LM